go.mod

```diff
@@ -43,9 +45,11 @@ require (
 )
 
 replace (
+	github.com/go-git/go-git/v5 => github.com/go-git/go-git/v5 v5.3.0
 	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega => github.com/onsi/gomega v1.5.0
 	k8s.io/api => k8s.io/api v0.0.0-20190918155943-95b840bb6a1f // kubernetes-1.16.0
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783 // kubernetes-1.16.0
```

Comment:
Quick question why do we have these new vendors now? Has something changed with the master merge? I remember a diff of only few files earlier. Just double checking that we don't have unwanted files.

Reply:
For me, even now the diff shows only `github.com/go-git/go-git/v5` and `k8s.io/apiextensions-apiserver` as new vendor packages added with no other changes.
Can you please recheck?

Reply:
I guess you didn't run `make revendor` after your changes? The vendor changes are a part of the merge master commit.

Reply:
It would be cleaner to split the commits into 
- Your changes
- Vendor changes
- Resolving conflicts with master (I prefer rebase with master instead of merge as it leads to one less commit changes)

Reply:
If you need help in doing this, we can do it over a call as well.

Reply:
I have the changes now on top of a rebase. Please check and confirm.

Reply:
Thanks for these changes. 
/lgtm